### PR TITLE
Adds coverage creation to github

### PR DESCRIPTION
**Pull request by alessioborraccino**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the changelog accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -12,10 +12,18 @@ on:
...
     runs-on: macos-latest
+    concurrency:
+      group: build-${{ github.head_ref }}
+      cancel-in-progress: true
     steps:
     - name: Display runner context
       run: hostname -s
-    - uses: actions/checkout@v2
+    - name: Display environment
+      run: echo $GITHUB_BASE_REF $GITHUB_HEAD_REF $GITHUB_HEAD_REF ${GITHUB_BASE_REF##*/} ${GITHUB_HEAD_REF##*/}
```
Comment (jose-cuadros-data4life) [inline .github/workflows/pr-test.yml:22]: GITHUB_HEAD_REF is twice

File: Package.swift [hunk 2]
```diff
@@ -3,6 +3,9 @@
 
 import PackageDescription
 
+let version = "0.6.0"
```
Comment (jose-cuadros-data4life) [inline Package.swift:6]: Will the version be set automatically?

File: .github/workflows/pr-test.yml (hunk 1 above)
Comment (alessioborraccino) [reply .github/workflows/pr-test.yml:22]: ahah will have to remove it anyways thanks

File: Package.swift (hunk 2 above)
Comment (alessioborraccino) [reply Package.swift:6]: for now no, but easier to update the package. i was just trying to remove the code smell

Comment (jose-cuadros-data4life) [reply Package.swift:6]: Yes, I liked both, this and the url string 👍

**Review by jose-cuadros-data4life — APPROVED** [review]

**Pull request merged**